raddb/mods-available/detail

```diff
@@ -100,7 +100,7 @@ detail {
 	#  locking:: Whether or not we should lock the detail file
 	#  before writing to it.
 	#
-	#  Mainly used if the detail file reader is be reading this file.
+	#  Mainly used if the detail file reader is to be reading this file.
```

Comment:
Mainly used if the detail file reader will be reading this file.

Reply:
'will be' sounds too certain. what if it's not going to be reading the file just yet? 
I chose 'is to be' since the previous wording suggested that was the intention of the text :)